bindings/nodejs/examples/21-collect-outputs.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * This example sends IOTA tokens with an expiration date and collects them.
+ */
+
+const getUnlockedManager = require('./account-manager');
+
+let manager
+let bob
+async function run() {
+    try {
+        manager = await getUnlockedManager();
+        const alice = await manager.getAccount('Alice');
+        bob = await manager.getAccount('Bob');
+
+        manager.listen(['NewOutput'], handleNewOutput)
+
+        const recipientAddress = bob.meta.publicAddresses[0].address
+        const amount = '1000000';
+
+        const outputData = await alice.prepareOutput(
+            {
+                recipientAddress,
+                amount,
+                unlocks: {
+                    expiration: {
+                        unixTime: new Date().getSeconds() + 15000
+                    }
+                }
+            },
+        );
+
+        const resp = await alice.sendOutputs([outputData])
+        console.log('Transaction is sent', resp)
```

Comment:
I think you need something like
```suggestion
        console.log('Transaction is sent', resp)
        // Sync bob account after 10 seonconds, so it detects the new output and the event gets emitted
        setTimeout(() => {
            await bob.sync()
        }, 10000);
```

Reply:
Added that one in the event handler.

Reply:
But syncing needs to be called to get the event in the first place